src/main/scala/learnshapeless/HLists.scala

```diff
@@ -32,17 +32,23 @@ object HLists extends App {
     (a) start the `sbt console`, and enter `:type learnshapeless.HLists.ex_newton`
     (b) if you are using Intellij, you can focus cursor on the expression and activate TypeInfo command (Ctrl-Shift-P on my Mac)
      */
-  def ex_newton = ???
+  def ex_newton: String :: Int :: Country :: HNil =
+    "Newton" :: 1642 :: England :: HNil
+  println(s"ex_newton $ex_newton")
 
   import shapeless.syntax.std.tuple._
   def eg_tuple_prepend_append: (String, String, Int, Country, Discovery) =
     ("Albert" +: eg1_einstein).tupled :+ TheoryOfRelativity
 
   /* Exercise 3: Prepend the String "Isaac" to `ex_newton` */
-  def ex_prepend = ???
+  def ex_prepend: String :: String :: Int :: Country :: HNil =
```

Comment:
I'm wondering if it's useful to get the attendees spell out the expected type in this exercise. I always do this out of habit, and also I think it will help the next person coming along that reads my code.

I'm not sure if it's the habit of every Scala developer though.